index.html

```diff
@@ -841,10 +841,10 @@ <h4 class="footer-widget__title">Developers</h4>
                             <h4 class="footer-widget__title">Contribute</h4>
                             <ul class="list-unstyled">
                                 <li>
-                                    <a href="https://github.com/kelvinkamau/Vibranium/issues/new?template=documentation-bug.md" target="_blank" rel="noopener noreferrer">File a bug</a>
+                                    <a href="https://github.com/DSC-Unilag/dsc-unilag.github.io/issues/new?template=documentation-bug.md" target="_blank" rel="noopener noreferrer">File a bug</a>
```

Comment:
```suggestion
                                    <a href="https://github.com/DSC-Unilag/dsc-unilag.github.io/issues/new" target="_blank" rel="noopener noreferrer">File a bug</a>
```

You can remove this part for now `?template=documentation-bug.md` since we do not have the file